package.json

```diff
@@ -43,7 +43,9 @@
   "dependencies": {
     "@ant-design/colors": "^6.0.0",
     "@ant-design/icons": "^4.2.1",
-    "@aws-amplify/core": "^3.8.21",
+    "@aws-amplify/auth": "^4.3.18",
```

Comment:
you can remove the `"aws-amplify"` import on line 60

Reply:
actually, you still need it in the dynamic import in `src/pages/_app.jsx`, so you can't remove it